.github/workflows/check_zonedata.yml

```diff
@@ -0,0 +1,24 @@
+name: Run check_zonedata
+
+# Workflow will run everytime someone pushes to the repo
+on: push
+
+jobs:
+
+  Run:  
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    - name: Check out code
+      uses: actions/checkout@v3
+      
+      # Needed to run the script
+    - name: Install bind-utills & upgrade
+      run: |
+          sudo apt-get install bind9-utils 
+          sudo apt upgrade bind9-utils
```

Comment:
Is it necessary to upgrade this package right after installing it?

Reply:
Ah no, for some reason i had to the first time i used it, but works fine now so i will remove it :)

Reply:
Gone :+1: